src/views/RecIM/views/Admin/index.jsx

```diff
@@ -54,6 +73,12 @@ const Admin = () => {
   const [openConfirmDeleteSurface, setOpenConfirmDeleteSurface] = useState();
   const [openConfirmDeleteSport, setOpenConfirmDeleteSport] = useState();
   const [snackbar, setSnackbar] = useState({ message: '', severity: null, open: false });
+  const [adminMenuAnchorEl, setAdminMenuAnchorEl] = useState();
+  const openAdminMenu = Boolean(adminMenuAnchorEl);
+  const [selectedDateIn, setSelectedDateIn] = useState(null);
+  const [selectedDateOut, setSelectedDateOut] = useState(null);
+  const [openRecimReportBox, setOpenRecimReportBox] = useState(null);
+  const [recimReport, setRecimReport] = useState(null);
 
```

Comment:
```suggestion
  const [selectedDateIn, setSelectedDateIn] = useState();
  const [selectedDateOut, setSelectedDateOut] = useState();
  const [openRecimReportBox, setOpenRecimReportBox] = useState();
  const [recimReport, setRecimReport] = useState();

```

empty useStates are considered null by default

Reply:
The date use states seem to have to be set null, they got set to the current date and time by default if I didn't give an argument.  I'll change the other two though.